rmc/utils/constraint.py

```diff
@@ -788,157 +768,15 @@ def get_rescue_1break_transcripts(
-        output_ht_path=f"{TEMP_PATH_WITH_FAST_DEL}/rescue_simul_chisq.ht",
-        overwrite=overwrite,
-    )
-
-    ht = hl.read_table(f"{TEMP_PATH_WITH_FAST_DEL}/rescue_simul_chisq.ht")
-    ht = get_max_chisq_per_group(ht, "section", "max_chisq")
-    ht = ht.filter(
-        (ht.max_chisq == ht.section_max_chisq)
-        & (ht.section_max_chisq < initial_threshold)
-        & (ht.section_max_chisq >= rescue_threshold)
-        & ~hl.literal(single_break_rescue_sections).contains(ht.section)
-    )
-    results_path = simul_search_round_bucket_path(
-        is_rescue=True,
-        search_num=1,
-        bucket_type="final_results",
-    )
-    ht = ht.checkpoint(f"{results_path}/merged.ht", overwrite=overwrite)
-    return ht.aggregate(hl.agg.collect_as_set(ht.section))
-
-
-def get_rescue_transcripts_and_create_no_breaks_he(
-    overwrite: bool, chisq_thresholds: Dict[str, Dict[str, float]] = CHISQ_THRESHOLDS
-) -> None:
-    """
-    Get transcripts found in rescue pipeline and write final no breaks HailExpression.
-
-    Function gets transcripts found in rescue single and simultaneous breaks searches
-    and creates final HT with all transcripts that do not have evidence of RMC.
-
-    .. note::
-        - Assumes top level keys in `chisq_threshold` dictionary are "initial"
-        and "rescue"
-        - Assumes nested keys in `chisq_threshold` dictionary are "single" and
-        "simul"
-
-    :param overwrite: Whether to overwrite output data if it exists.
-    :param chisq_thresholds: Dictionary of chi square significance thresholds.
-        Default is CHISQ_THRESHOLDS.
-    :return: None; function writes HailExpression to resource path.
-    """
-    # Get the sections (transcript_start_stop) found in initial simultaneous breaks search,
-    # rescue single break search, and rescue simultaneous breaks search
-    init_simul_sections, rescue_single_sections = get_rescue_1break_transcripts(
-        overwrite=overwrite,
-        rescue_threshold=chisq_thresholds["rescue"]["single"],
-    )
-    rescue_simul_sections = get_rescue_2breaks_transcripts(
-        overwrite=overwrite,
-        initial_threshold=chisq_thresholds["initial"]["simul"],
-        rescue_threshold=chisq_thresholds["rescue"]["simul"],
-    )
-    sections_with_breaks = init_simul_sections.union(rescue_simul_sections).union(
-        rescue_single_sections
-    )
-
     # Read in the no break found HT from initial search round 1
     ht = hl.read_table(
         single_search_round_ht_path(
-            is_rescue=False,
             search_num=1,
             is_break_found=False,
             is_breakpoint_only=False,
         )
     )
-    ht = ht.filter(~hl.literal(sections_with_breaks).contains(ht.section))
+    ht = ht.filter(~hl.literal(simul_sections).contains(ht.section))
     no_break_sections = ht.aggregate(hl.agg.collect_as_set(ht.section))
     logger.info(
         "%i transcripts did not have any evidence of RMC", len(no_break_sections)
```

Comment:
The `calculate_section_chisq` section at line 863 needs to be deleted, must have been a copy-paste mistake (not sure how that happened)

Reply:
closing this since you've deleted the extra function